models/intel/instance-segmentation-security-0002/description/instance-segmentation-security-0002.md

```diff
@@ -0,0 +1,45 @@
+# instance-segmentation-security-0002
+
+## Use Case and High-Level Description
+
+This model is an instance segmentation network for 80 classes of objects.
+It is a Mask R-CNN with ResNet50 backbone, FPN, RPN, detection and
+segmentation heads.
+
+## Example
+
+![](./instance-segmentation-security-0002.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| MS COCO val2017 box AP          | 40.8%                                     |
+| MS COCO val2017 mask AP         | 36.9%                                     |
+| Max objects to detect           | 100                                       |
+| GFlops                          | 423.0842                                  |
+| MParams                         | 48.3732                                   |
+| Source framework                | PyTorch\*                                 |
+
+Average Precision (AP) is defined and measured according to standard
+[MS COCO evaluation procedure](http://cocodataset.org/#detection-eval).
+
+## Performance
```

Comment:
we are removing empty sections from documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please do this as well

Reply:
Fixed.